README.md

```diff
@@ -210,6 +211,27 @@ For dex.decred.org, reach out
 [on Element](https://matrix.to/#/!mlRZqBtfWHrcmgdTWB:decred.org?via=decred.org&via=matrix.org&via=planetdecred.org)
 to appeal.
 
+## Fees
+
+The dex does not charge trading fees, but users have to pay on-chain transaction
+fees. Unlike on centralized exchanges where users can trade any arbitrary amount
+of funds, orders on the dex must be made in increments of a specified lot size.
+For example, if the lot size for the DCR/BTC market is 40 DCR, users can
+make orders to buy or sell DCR in increments of 40.
+
+You might be lucky and have your entire order matched by one other user. In
+this case, you will have to pay 2 transaction fees: one for sending funds on
```

Comment:
The order can match multiple users and still only require 2 transactions. Matches can be grouped. You are correct in your description of the conditions necessary for the best case scenario, but the benefit you describe (2 transactions) is not unique to that particular mode. In fact, at the API level, there is almost no concept of a transaction. All we should really say is that **the best case scenario is when the entire order is consumed by a single match**, in which case only one swap and one redemption will be created, and the worst-case scenario is **one single-lot match per epoch**.

Reply:
If you match with two separate users, won't you have to pay for two separate redemptions?

Reply:
Agree with best case scenario 100%.  Worst-case I think is more nuanced than that, esp for taker where it's still potentially worst case with multiple matches in the same epoch.